sos/plugins/samba.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
         self.add_cmd_output([
             "wbinfo --domain='.' -g",
             "wbinfo --domain='.' -u",
+            "testparm -s",
             "testparm -s -v"
```

Comment:
Is there any use in collecting `-s -v`, in that case? It seems it would just record additional data that is pre-determined for the installed version of Samba?